src/Hooks/useStore.js

```diff
@@ -1,20 +1,17 @@
 import create from 'zustand';
-import {persist} from 'zustand/middleware';
 
 const useStore = create(
-	persist(
-		set => ({
-			entries: [],
-			addEntry: entry => {
-				set(state => {
-					return {entries: [...state.entries, entry]};
-				});
-			},
-		}),
-		{
-			name: 'Entries', // unique name
-			getStorage: () => localStorage,
-		}
-	)
+	set => ({
+		entries: [],
+		addEntry: entry => {
+			set(state => {
+				return {entries: [...state.entries, entry]};
+			});
+		},
+	}),
+	{
+		name: 'Entries', // unique name
+		getStorage: () => localStorage,
+	}
```

Comment:
This is part of persist and therefore should be deleted. Additionally, I strongly suggest to delete all data in the local storage of your browser. So there won't be any conflicts in the future.
```suggestion

```